### PR TITLE
Revising Running section

### PR DESCRIPTION
**Pull request by giovannipizzi**

### CODE REVIEW

**Comment by ramirezfranciscof** [comment]
Thanks for the revision! These changes are ok by me, not sure if you want me to approve it or if we should maybe leave that to @chrisjsewell after he reviewed it himself.

**Review by chrisjsewell — APPROVED** [review]

**Comment by chrisjsewell** [comment]
> These changes are ok by me, not sure if you want me to approve it or if we should maybe leave that to @chrisjsewell after he reviewed it himself.

yep look goot to me 👍

**Pull request merged**
